graph-commons/src/main/scala/ch/datascience/http/server/HttpServer.scala

```diff
@@ -26,15 +26,16 @@ import ch.datascience.controllers.InfoMessage._
 import org.http4s.server.blaze._
 import org.http4s.{HttpRoutes, Request, Response, Status}
 
+import scala.concurrent.ExecutionContext
 import scala.language.higherKinds
 
 class HttpServer[F[_]: ConcurrentEffect](
     serverPort:    Int,
     serviceRoutes: HttpRoutes[F]
-)(implicit timer:  Timer[F]) {
+)(implicit timer:  Timer[F], executionContext: ExecutionContext) {
 
   def run: F[ExitCode] =
-    BlazeServerBuilder[F]
+    BlazeServerBuilder[F](executionContext)
```

Comment:
Does executionContext have to be passed explicitly to the Builder?

Reply:
Yes the apply method with implicit execution context is now deprecated.